pkgs/ffigen/lib/src/visitor/fix_overridden_methods.dart

```diff
@@ -128,7 +128,8 @@ class FixOverriddenMethodsVisitation extends Visitation {
     for (final method in node.methods) {
       if (method.isClassMethod) continue;
       final (root, rootMethod) = _findRootWithMethod(node, method);
-      if (method.isProperty == rootMethod.isProperty) continue;
+      if ((method.kind == ObjCMethodKind.propertyGetter) ==
```

Comment:
So XNOR? Maybe a comment like:

// If `method` are `rootMethod` are of the same kind then there is nothing to do.